### ISSUE TITLE
Upgrading from 2.0.5 to 2.0.6 breaks isomorphic/universal/node apps

### ISSUE THREAD

**Issue by Stanback** [bramstein/fontfaceobserver#88]
After upgrading I'm getting an error about document being undefined when executing my app with Node.js. I'm importing fontfaceobserver and then invoking it on componentDidMount.

```
18:42:36 1|react    | ReferenceError: document is not defined
18:42:36 1|react    |     at /usr/src/app/node_modules/fontfaceobserver/fontfaceobserver.standalone.js:1:21
18:42:36 1|react    |     at Object.<anonymous> (/usr/src/app/node_modules/fontfaceobserver/fontfaceobserver.standalone.js:7:362)
18:42:36 1|react    |     at Module._compile (module.js:573:32)
18:42:36 1|react    |     at Object.Module._extensions..js (module.js:582:10)
18:42:36 1|react    |     at Module.load (module.js:490:32)
18:42:36 1|react    |     at tryModuleLoad (module.js:449:12)
18:42:36 1|react    |     at Function.Module._load (module.js:441:3)
18:42:36 1|react    |     at Function._load (/usr/lib/node_modules/pm2/node_modules/pmx/lib/transaction.js:62:21)
18:42:36 1|react    |     at Module.require (module.js:500:17)
18:42:36 1|react    |     at require (internal/module.js:20:19)
```

I'm guessing there was a check for typeof document !== undefined which got removed somewhere.

**Comment by bramstein** [comment]
Looks like the Closure compiler rewrote some code so that `document` is used in the top-level function. Why is `document` undefined for you?

**Comment by Stanback** [comment]
@bramstein I have a component that's shared between Node and the browser. It gets rendered initially on the server using Node.js (which doesn't have document defined) and then handed over to the client.

I built a fresh fontfaceobserver from source just now (with `grunt dist`) and it looks like the version of closure compiler that's specified by grunt-closurecompiler (version 1.6.1) is encapsulating document correctly and is working for me again.

**Comment by bramstein** [comment]
@Stanback Odd, I thought I was running the latest. Re-installing made the problem go away. I released 2.0.7, which seems to fix the issue.

**Comment by Stanback** [comment]
Awesome, thanks! 👍
